test/uk/gov/hmrc/cipphonenumberverification/WiremockSupport.scala

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
there is a wiremock support class provided by uk.gov.hmrc.http.test

Reply:
Thanks for this didn't realise there was one. Only thing is thing protocol is hard-corded. But not an issue